contracts/Oracle.sol

```diff
@@ -144,6 +173,44 @@ contract Oracle is Ownable {
         oracleDisputePeriod[_oracle] = _disputePeriod;
 
         emit OracleDisputePeriodUpdated(_oracle, _disputePeriod);
-        (_oracle, _disputePeriod);
+    }
+
+    /**
+     * @notice set batch underlying asset price
+     * @dev underlying price can only be set after locking period is over and before starting dispute period
+     * @param _batch (hash of underlying, stike, collateral and expiry)
+     * @param _expiryTimestamp batch expiry timestamop
+     * @param _roundsBack number of chainlink price feed roundback
+     */
+    function setBatchUnderlyingPrice(
+        bytes32 _batch,
+        uint256 _expiryTimestamp,
+        uint256 _roundsBack
+    ) external onlyController {
+        AggregatorInterface oracle = AggregatorInterface(batchOracle[_batch]);
+
+        require(address(oracle) != address(0), "Oracle: no oracle for this specific batch");
+        require(isLockingPeriodOver(_batch, _expiryTimestamp), "Oracle: locking period is not over yet");
+        require(batchPriceAt[_batch][_expiryTimestamp].timestamp == 0, "Oracle: dispute period started");
+
```

Comment:
```suggestion
        require(batchOracle[_batch] != address(0), "Oracle: no oracle for this specific batch");
        require(isLockingPeriodOver(_batch, _expiryTimestamp), "Oracle: locking period is not over yet");
        require(batchPriceAt[_batch][_expiryTimestamp].timestamp == 0, "Oracle: dispute period already started");
       
        AggregatorInterface oracle = AggregatorInterface(batchOracle[_batch]);
```

I feel like it's a bit better to have all the require statements first, and also not use address(oracle) to convert it back from AggregatorInterface instance